extension.json

```diff
@@ -23,6 +23,7 @@
     "email": "data@open-contracting.org"
   },
   "testDependencies": [
-    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/master/extension.json"
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/master/extension.json",
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_awardCriteria_extension/master/extension.json"
```

Comment:
The award criteria extension should be in `dependencies` since this extension `$ref`'erences a definition from it, per the [handbook](https://ocds-standard-development-handbook.readthedocs.io/en/latest/extensions/index.html#create-an-extension).

Reply:
According to the handbook, only if this extension adds fields to a definition from the award criteria extension, so I think that can be removed from `testDependencies`.